src/components/CurrentUser/CurrentUser.jsx

```diff
@@ -1,12 +1,69 @@
 import React from 'react';
+
+import { CurrentUserPropTypes } from '../PropTypes/CurrentUserPropTypes';
 import './CurrentUser.scss';
 
-export const CurrentUser = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+export class CurrentUser extends React.Component {
+  state={
+    user: 0,
+  }
+
+  componentDidMount() {
+    this.loadData();
+  }
+
+  componentDidUpdate(prevProps) {
+    if (prevProps.userId !== this.props.userId) {
+      this.loadData();
+    }
+  }
+
+  clearUser = () => {
+    this.props.selectUser(0);
+  }
+
+  async loadData() {
+    const user = await (this.props.getUser(this.props.userId));
```

Comment:
рекомендую не оборачивать
```suggestion
    const user = await this.props.getUser(this.props.userId);
```